public/creators-landing/src/sections/main-section/index.js

```diff
@@ -26,13 +26,18 @@ const logAction = (action, value) => {
 export const MainHome = () => {
   return (
     <GradientBackground align="center">
-      <Container animation="fadeIn" role="main">
+      <Container role="main">
```

Comment:
Removed the animation here because it was slowing down the `First Input Delay (FID)` and `Estimated Input Latency`. The other pages fade in so I think we're good here.